projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+# Reuse places & organizers
+
+When creating an event via the UiTdatabank Entry API, you can and must use existing places (locations) and organizers in UiTdatabank.
+
+By reusing existing places and organizers you save yourself work, as you don't have to create the place or organizer yourself. In addition, you ensure that no duplicates are created in UiTdatabank and that other integrators can also use the same unique and clean dataset.
+
+Only in the case a certain place or organizer does not exist yet, a new one can be created.
+
+<!-- theme: warning -->
+
+> When sending event content to UiTdatabank trough the Entry API it is essential to reuse existing organizer IDs and place IDs. Otherwise your integration can be blocked because of the creation of duplicates.
```

Comment:
```suggestion
> When sending event content to UiTdatabank trough the Entry API you **must** reuse existing organizer IDs and place IDs when available. Otherwise your integration can be blocked because of the creation of duplicates.
```

Feels important enough to make it a strict requirement?